railgun/internal/mgrutils/workspace.go

```diff
@@ -0,0 +1,39 @@
+package mgrutils
+
+import (
+	"context"
+	"fmt"
+	"github.com/kong/go-kong/kong"
+)
+
+// TODO these functions have been copied from 1.x cli/ingress-controller/util.go
+// https://github.com/Kong/go-kong/pull/56 provides go-kong replacements for them,
+// but is not yet released
```

Comment:
I have just cut go-kong v0.18.0 with that functionality included. Please go ahead and use that already.